### PR TITLE
Created a card profile result component

### PR DESCRIPTION
**Pull request by cph-kiwi**
# Description

I created a component to display the results of the quiz, to be able to see different results depending on the answers that a user gives in the quiz. This is so that a user can see the guidance bullet points for their specific result.

Fixes # 16
https://github.com/HackYourFuture-CPH/Guide-IT/issues/16

# How to test?

Open the branch and check the component out in storybook. The component is modelled in two ways, as a personal profile and as a professional profile.

# Checklist

- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by qamarfj — APPROVED** [review]
looks good

**Review by aijazraja — COMMENTED** [review]
Hi Beth
The List item part of the professional and personal profile should not a little different in colors? because if you see in the personal profile area it is more white than a professional one.

**Comment by wwang107** [comment]
Looking good to me. Nice that you use `flex`, it make life easier.

**Review by wwang107 — APPROVED** [review]
LGTM

**Review by wwang107 — CHANGES_REQUESTED** [review]
Since someone mention about the background color, could you add a prop that dynamcally change the background color?

**Comment by cph-kiwi** [comment]
I have had a look at the background colour. It is actually the same in both cases. However an optical illusion happens depending on what the cards are placed on top of that makes them look different to each other. Here is an article about it. Check out the last paragraph and image (square of different shades of grey where the grey colour in the middle of each box is the same colour). https://blog-college.ku.edu/visual-illusion-are-you-teamwhitegold-or-teamblueblack/

File: src/client/components/CardProfileResult/CardProfileResultComponent.styles.css [hunk 1]
```diff
@@ -1,30 +1,45 @@
...
+.card-profile-result-list-items li::before {
+  content: ' ';
+  width: 14px;
+  height: 14px;
+  background-color: #e95328;
+  display: block;
+  position: absolute;
+  border-radius: 9999px;
+  left: -40px;
+  top: calc(50% - 7px); /* 7px is half the height */
```
Comment (TITHI1244) [inline src/client/components/CardProfileResult/CardProfileResultComponent.styles.css:37]: You have nicely designed the bullet point. But do we not need to use images instead from figma? What do you think?

**Review by TITHI1244 — APPROVED** [review]
Very nicely done. Working perfectly as intended.

**Review by wwang107 — APPROVED** [review]
Nicely done. Good job

**Review by aijazraja — APPROVED** [review]
Good Job

**Pull request merged**